### DISCUSSION TITLE
How to set custom permissions

### DISCUSSION THREAD

**Discussion by Mayanktaker** [directus/directus#14380] [Archive]
Hello. I come from Wordpress where I can set custom permission for roles. Like author1 role can edit only content which is created by author1 role user. And author2 role can do the same with only author2 role. So they can see content but can not edit or deleted each others content. 

I can make 2 similar roles in Directus but can not change permission like wordpress. How can I set permission to roles to let them only see each others content but can not modify others content ?

**Comment by br41nslug** [comment]
To limit who can edit what you can "use custom" under roles & permissions.
![image](https://user-images.githubusercontent.com/9389634/178483422-7618e09a-60b5-4fa9-b24f-4e14b0e841bc.png)
Here you can define custom rules per CRUD (create/read/update/delete) operation. Using your example you'd probably want to go with something like `User Created -> Role Equals $CURRENT_ROLE` for the `create` and `update` operations.
![image](https://user-images.githubusercontent.com/9389634/178483856-ad33dd99-50e1-4e72-be8d-3271f9ffcb5d.png)
https://docs.directus.io/configuration/users-roles-permissions/#configure-permissions
